server/apps/immich/src/api-v1/asset/dto/add-assets-to-shared-link.dto.ts

```diff
@@ -1,6 +1,6 @@
 import { IsNotEmpty } from 'class-validator';
 
-export class UpdateAssetsToSharedLinkDto {
+export class AddAssetToSharedLinkDto {
   @IsNotEmpty()
-  assetIds!: string[];
+  ids!: string[];
```

Comment:
We should maybe consider using a single dto for adding assets in all our endpoints. They should ideally work the same way. Same with bulk remove assets. Basically here and adding to an album.

Reply:
No, I mean that this seems really similar to adding and removing assets from an _album_. In fact, the DTOs are basically exactly the same for adding to a shared link as to adding to an album. Can we use a single "AddAssetsDto" in both places?

Reply:
Got it, sounds good!